core/commands/add.go

```diff
@@ -14,6 +14,7 @@ import (
 	dag "github.com/ipfs/go-ipfs/merkledag"
 	dagtest "github.com/ipfs/go-ipfs/merkledag/test"
 	mfs "github.com/ipfs/go-ipfs/mfs"
+	cide "github.com/ipfs/go-ipfs/thirdparty/cidextra"
```

Comment:
I assume this is temporary.

Reply:
More or less, I would like to get something working first, merge this p.r. and then worry about finding a better place for the two new packages in thirdparty.